include/kos/thread.h

```diff
@@ -224,14 +221,12 @@ typedef struct kthread_attr {
 
 /** \defgroup thd_modes             Threading system modes
 
-    The threading system will always be in one of the following modes. This
-    represents the type of scheduling done by the system (or the special case of
-    threads not having been initialized yet).
+    The threading system will always be in one of the following modes. This 
+    represents either pre-emptive scheduling or an un-initialized state.
 
     @{
 */
 #define THD_MODE_NONE       -1  /**< \brief Threads not running */
-#define THD_MODE_COOP       0   /**< \brief Cooperative threading mode */
```

Comment:
This should probably stay too, but with the comment saying that it isn't supported anymore.